pkg/node/raft_store_test.go

```diff
@@ -24,32 +26,34 @@ func Test_StoreInMemOpenSingleNode(t *testing.T) {
 	eciesKey, err := ecies.GenerateKey(rand.Reader, secp256k1.S256(), nil)
 	require.NoError(t, err)
 
-	key := CosignerEd25519Key{
+	key := pcosigner.CosignerEd25519Key{
 		PubKey:       dummyPub,
 		PrivateShard: []byte{},
 		ID:           1,
 	}
 
-	cosigner := NewLocalCosigner(
+	cosigner := pcosigner.NewLocalCosigner(
 		log.NewNopLogger(),
-		&RuntimeConfig{},
-		NewCosignerSecurityECIES(
-			CosignerECIESKey{
+		&pcosigner.RuntimeConfig{},
+		pcosigner.NewCosignerSecurityECIES(
```

Comment:
doesn't need to be done in this PR, but with the pkg named `cosigner` we can remove the `Cosigner` stutter from the types and methods. e.g. `NewCosignerSecurityECIES` -> `cosigner.NewSecurityECIES` and `CosignerECIESKey` to `cosigner.ECIESKey`, etc.

Reply:
You read my mind :)
That was my idea.

In addition to change the names of the variables to ´cosign´ instead of 'cosigner' 

My suggestion is to do it in next PR (I am already on it).